packages/webpack.config.js

```diff
@@ -87,16 +85,14 @@ function getReactMaterialExternals() {
   ));
 }
 
-function getMaterialExternals() {
+function getMaterialExternals(withReact) {
```

Comment:
where is this withReact being passed as true/false? Is this change unintentional?